backend/molgenis-emx2-typescript/build.gradle

```diff
@@ -8,5 +8,6 @@ dependencies {
 task generateTypes(type: JavaExec) {
     group = 'application'
     classpath = sourceSets.main.runtimeClasspath
+    workingDir = rootProject.projectDir
```

Comment:
this was necessary to generate the types (there was a bug created in the previous refactoring of the generateTypes task). This ensures the path is interpreted relatively to the root of the molgenis-emx2 repo.

Reply:
I also updated docs of nuxt3-ssr to explain how to run that task routinely.

Reply:
you probably used another command than I did. What command do you use? because then we should add that maybe to the docs? Unless you also see this as an improvement?